docs/hub/datasets-file-names-and-splits.md

```diff
@@ -0,0 +1,146 @@
+# File names and splits
+
+To host and share your dataset, create a dataset repository on the Hugging Face Hub and upload your data files.
+
+This guide will show you how to name your fiels and directories in your dataset repository when you upload it and enable all the Dataset Hub features like the Dataset Viewer.
+A dataset with a supported structure and file format (`.txt`, `.csv`, `.parquet`, `.jsonl`, `.mp3`, `.jpg`, `.zip` etc.) automatically has a dataset viewer on its dataset page on the Hub.
+
+Note that you can also define your own custom structure, see the documentation on [Manual Configuration](./datasets-manual-configuration) for more information
+
+## Main use-case
+
+The simplest dataset structure has two files: `train.csv` and `test.csv` (this works with any supported file format).
```

Comment:
I would say that it's simpler to only have 1 data file, ie. `data.csv`. In which case it will be considered as the train split. Maybe mention this first, no?

Reply:
The train.csv and test.csv case is explained later.